src/Microsoft.Data.SqlClient/src/Resources/Strings.de.resx

```diff
@@ -4737,4 +4737,7 @@
   <data name="SQL_RemoteCertificateNotAvailable" xml:space="preserve">
     <value>Das Zertifikat ist während der Überprüfung des Zertifikats nicht verfügbar.</value>
   </data>
+  <data name="AAD_FIC_Invalid_Setup" xml:space="preserve">
```

Comment:
There's no need to provide any non-English strings. The English strings are translated via an automated localization process after the PR is merged to main.

Reply:
Non-English strings removed